pkg/cloud/cloud.go

```diff
@@ -56,6 +56,7 @@ var (
 		VolumeTypeST1,
 		VolumeTypeStandard,
 	}
+	VolumeNotBeingModified = fmt.Errorf("volume is not being modified")
```

Comment:
Could you move this down below, together with other errors?

Reply:
fixed. good catch.